getconf/base.py

```diff
@@ -169,6 +170,19 @@ def gettimedelta(self, key, default='0d', doc=''):
         else:
             return datetime.timedelta(**{unit: value})
 
+    def getpath(self, key, default=Path(), doc=''):
+        assert (
+            default is None or isinstance(default, Path)
+        ), 'getpath("%s", %r) has an invalid default value type.' % (key, default)
+        value = self._get(key, default=default, doc=doc, type_hint='Path')
```

Comment:
for the `type_hint`, `path` lowercase? `pathlib.Path`?

Reply:
I'd say `pathlib.Path` to be explicit but I don't care much, really.